sunpy/map/mapbase.py

```diff
@@ -317,6 +318,26 @@ def _repr_html_(self):
         count_nan = np.isnan(self.data).sum()
         count_inf = np.isinf(self.data).sum()
 
+        # If data is from a dask array, pull data into memory for quicklook() method
+        # For basic html representation, only use dask html representation without computing data
+        if isinstance(finite_data, DaskArray):
```

Comment:
See above comment too:

```suggestion
        if DASK_INSTALLED and isinstance(finite_data, DaskArray):
```

Reply:
Added changes as suggested